bin/simput-cli.js

```diff
@@ -112,15 +112,34 @@ if (program.input && program.output && !program.gui) {
         }
     })
     .post(function(req, res) {
-        //receive new file content and update the file at program.input
+
+        /* Results */
+
         for (var key in req.body.results) {
             mkdir('-p', toAbsolutePath(path.dirname(path.join(program.output, key))));
             fs.writeFileSync(path.join(program.output, key), req.body.results[key]);
         }
 
-        //save the model too
+        /* Copies */
+
+        if (typeof(req.body.copies) !== 'undefined') {
```

Comment:
I would do `if (req.body.copies) {` instead of your complex condition that won't work with a null.

Reply:
Fixed with commit 270c70e8eaafb48664c9f6fe99e000c069962240.